benchmarks/graphics_pipeline/main.cpp

```diff
@@ -189,6 +212,150 @@ void ProjApp::Setup()
         mCamera.SetPerspective(60.f, GetWindowAspect());
     }
 
+    // Texture image, view, and sampler
+    {
+        // SkyBox
+        {
+            grfx_util::ImageOptions options = grfx_util::ImageOptions().MipLevelCount(PPX_REMAINING_MIP_LEVELS);
+            PPX_CHECKED_CALL(grfx_util::CreateImageFromFile(GetDevice()->GetGraphicsQueue(), GetAssetPath("basic/models/spheres/basic-skybox.jpg"), &mImage, options, true));
+
+            grfx::SampledImageViewCreateInfo viewCreateInfo = grfx::SampledImageViewCreateInfo::GuessFromImage(mImage);
+            PPX_CHECKED_CALL(GetDevice()->CreateSampledImageView(&viewCreateInfo, &mSampledImageView));
+        }
+
+        // Sampler
+        grfx::SamplerCreateInfo samplerCreateInfo = {};
+        samplerCreateInfo.magFilter               = grfx::FILTER_LINEAR;
+        samplerCreateInfo.minFilter               = grfx::FILTER_LINEAR;
+        samplerCreateInfo.mipmapMode              = grfx::SAMPLER_MIPMAP_MODE_LINEAR;
+        samplerCreateInfo.minLod                  = 0;
+        samplerCreateInfo.maxLod                  = FLT_MAX;
+        PPX_CHECKED_CALL(GetDevice()->CreateSampler(&samplerCreateInfo, &mSampler));
+    }
+
+    // Meshes
+    {
+        // SkyBox
+        TriMesh  mesh = TriMesh::CreateCube(float3(1, 1, 1), TriMeshOptions().TexCoords());
+        Geometry geo  = {};
+        PPX_CHECKED_CALL(Geometry::Create(GeometryOptions::InterleavedU16().AddTexCoord(), mesh, &geo));
+        PPX_CHECKED_CALL(grfx_util::CreateMeshFromGeometry(GetGraphicsQueue(), &geo, &mSkyBox.mesh));
+
+        // Sphere
+        mesh = TriMesh::CreateSphere(/* radius = */ 1, /* longitudeSegments = */ 10, /* latitudeSegments = */ 10, TriMeshOptions().VertexColors());
```

Comment:
nit: you could use another anonymous scope for the sphere.
This way the pattern is exactly the same and you don't reuse variables across scopes.

Reply:
same below if you decide to apply this change.

Reply:
I tried to apply this, but I didn't find a way to configure the `.clang-format` file so that I can properly format nested braces. When I attempt to format the code, it still doesn't handle nested braces correctly.
```
{
    // comment 1
    {
        // code 1 goes here
    }
    // comment 2
    {
        // code 2 goes here
    }
}
```
It formats in an unexpected way. However, there is a way to make this work — by adding a piece of code without nested braces, but this is not the case here.

Reply:
You don't need to nest, see the suggested edit. It's only so the tmp variable remains very local.